modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc

```diff
@@ -0,0 +1,108 @@
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create an {rh-rhacm} policy in the Hub cluster.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CR, and then save the YAML in the `cgu-cr.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4915
+  namespace: ztp-group-du-sno
+spec:
+  deleteObjects: true
+  clusters: <1>
+  - cnfdb1
+  - cnfdb2
+  enable: true
+  managedPolicies:
+  - du-upgrade-platform-upgrade <2>
+  remediationStrategy:
+    maxConcurrency: 2 <3>
+    timeout: 240 <4>
+----
+<1> The list of clusters to update.
+<2> The name of the policy to apply.
+<3> The `maxConcurrency` field signifies the number of clusters updated at the same time.
+<4> The update timeout in minutes.
+
+. Create the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f cgu-cr.yaml
+----
+
+.Verification
+
+. Check if the `ClusterGroupUpgrade` CR was created in the Hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get cgu-cr -A -w
+----
++
+.Example output
++
+[source,terminal]
+----
+NAMESPACE          NAME              AGE
+ztp-group-du-sno   du-upgrade-4915   5s
+ztp-install        cnfdb1            4d5h
+ztp-install        cnfdb2            4d5h
+----
+
+. Check the status of the policies by running the following command:
++
+[source,terminal]
+----
+$ oc get policies -A | grep upgrade
+----
++
+.Example output
+[source,terminal]
+----
+cnfdb1                ztp-group-du-sno.du-upgrade-platform-upgrade                   inform               Compliant          3d19h
+cnfdb2                ztp-group-du-sno.du-upgrade-4915-du-upgrade-platform-upgrade   enforce              Compliant          23s <1>
+cnfdb2                ztp-group-du-sno.du-upgrade-platform-upgrade                   inform               NonCompliant       3d19h
+ztp-group-du-sno      du-upgrade-4915-du-upgrade-platform-upgrade                    enforce              Compliant          24s <1>
+ztp-group-du-sno      du-upgrade-platform-upgrade                                    inform               NonCompliant       3d19h
+
+----
+<1> The status of the targeted clusters is changed to `enforce``.
+
+. Check the status of the update by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion,node
```

Comment:
I think this is misleading. This describes how to check the status of the platform upgrade that happens as a result of the managed policy being a platform upgrade policy.

I think we should make it clear here that:
1. In order to check the status of the CGU you need to look at either the output of `oc get cgu -n <namespace> <cgu_cr_name> -o jsonpath='{.status}'` or `the status part of c get cgu -n <namespace> <cgu_cr_name> -oyaml`
2. if one of the managed policies is of platform upgrade type, you can check the status of the upgrade by using the command at line 95.